cli/ballerina-cli/src/test/resources/test-resources/mockFunctionNative/modules/util2/tests/tes.bal

```diff
@@ -0,0 +1,11 @@
+import ballerina/test;
+
+import ballerina/os;
+
+@test:Config {}
+function intAddTest3() {
+    test:assertNotEquals(os:getUserHome(), "alpha");
+    test:assertNotEquals(os:getUserHome(), "beta");
+
```

Comment:
Can we use constants for "alpha" and "beta" as they are used multiple times in several tests?

Reply:
Those constants are defined and used across modules. and They are simple test cases. Do we need to modify them too?